battleship.py

```diff
@@ -310,7 +336,30 @@ def placeShip(data):
 Returns: None
 '''
 def clickUserBoard(data, row, col):

```

Comment:
Maintain a proper naming conventions for variables

Reply:
done